streaming_form_data/targets.py

```diff
@@ -107,6 +108,42 @@ def on_finish(self):
             self._fd.close()
 
 
+class DirectoryTarget(BaseTarget):
+    """DirectoryTarget writes (streams) the different input to an on-disk directory."""
+
+    def __init__(
+        self,
+        directory_path: str,
+        allow_overwrite: bool = True,
+        *args,
+        **kwargs
+    ):
+        super().__init__(*args, **kwargs)
+
+        self.directory_path = directory_path
+
+        self._mode = 'wb' if allow_overwrite else 'xb'
+        self._fd = None
+        self.multipart_filenames = []
+        self.multipart_content_types = []
+
+    def on_start(self):
+        self.multipart_filename = Path(self.multipart_filename).name
```

Comment:
This might lead to a vulnerability in case the client sends something like `../../path/to/passwd`.

One alternative here would be to ask the user to provide a prefix inside `__init__` and use that here to construct filesystem paths.

Reply:
I added a comment above this line to explain that Path().name only keeps the file name. In addition, I made a new unit test to verify this behavior.

Reply:
Nice, thank you.